spec/views/users/users_show_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'rails_helper'
+
+RSpec.describe 'user#Show', type: :feature do
+  describe 'User' do
+    before(:each) do
+      @user1 = User.create(name: 'Sammy', photo: 'sammy.png', bio: 'Sammy bio', posts_counter: 0,
+                           email: 'sammy@gmail.com', password: 'test12345')
+      @user2 = User.create(name: 'Ahmed', bio: 'Ahmed bio', photo: 'ahmed.png', email: 'ahmed@gmail.com',
+                           password: 'test12345')
+      @user3 = User.create(name: 'John', bio: 'John bio', photo: 'john.png', email: 'john@gmail.com',
+                           password: 'test12345')
```

Comment:
- [ ] currently, when I run these users-show tests on this page all of them fail. please make sure that these tests are working to ensure your application runs as expected.